CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [release-84] - 2019-08-19
```

Comment:
Anyone understand why this is not causing an automatic hyperlink like the other headings in this file are?

Reply:
I think you need to add the release to the list of links at the bottom of the document.

Reply:
Ah, thank you!

Reply:
(fixed it)